pkgs/by-name/wh/when/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenvNoCC
+, perl
+, installShellFiles
+, fetchFromBitbucket
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname   = "when";
+  version = "1.1.45";
+
+  src = fetchFromBitbucket {
```

Comment:
The indentation is not good.

Reply:
I appreciate your attention to details. However, I'd like to share my approach to reviews, especially concerning first-time contributors.

My primary goal is to guide these new contributors in addressing the most critical issues in their submissions. For aspects like coding style, which can be easily managed through a linter or are not strictly enforced by our CI processes, I often choose not to focus on them during the initial review stages. This approach is intentional – being overly meticulous on minor points can sometimes be **counterproductive**, potentially **discouraging** new contributors from ongoing participation.

In line with this, I usually refrain from making additional stylistic or non-critical comments post-merge unless they bring significant value or insights. This helps maintain a welcoming and constructive environment for all contributors, especially those who are just beginning their journey with Nix.